src/worker.rs

```diff
@@ -241,7 +241,10 @@ impl<T: Socket + ?Sized> Worker<T> {
     }
 
     fn send_packet(&self, packet: &Packet) -> Result<(), Box<dyn Error>> {
-        for _ in 0..self.duplicate_packets {
+        for i in 0..self.duplicate_packets {
+            if i > 0 {
+                std::thread::sleep(Duration::from_millis(1));
```

Comment:
Can we put `Duration::from_millis(1)` to a constant like `const DEFAULT_DUPLICATE_DELAY: Duration = Duration::from_millis(1);`?

Reply:
Yep the delay has been changed to the constant `DEFAULT_DUPLICATE_DELAY` thanks!